blocks/identity-block/components/SocialSignOn/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react';
+import useIdentity from '../Identity';
+import './styles.scss';
+
+const SocialSignOn = ({ onError, redirectURL }) => {
+  const { Identity } = useIdentity();
+  const config = Identity?.configOptions ?? {};
```

Comment:
This should be a `useState` variable -- I don't think this line is being executed on the re-render after the `fetchConfig` effect

```const [config, setConfig] = useState(() => Identity?.configOptions ?? {})```

Reply:
Thanks @mattkim93